.travis.yml

```
@@ -19,11 +19,12 @@ addons:
 install:
     - docker build -t aiida-prerequisites .
     - docker run -d aiida-prerequisites
+    - "export DOCKERID=`docker ps -qf 'ancestor=aiida-prerequisites'`"
+    - while [[ \"healthy\" != `docker inspect --format='{{json .State.Health.Status}}' $DOCKERID` ]]; do sleep 1; done
 
 # I just run a command with verdi that returns a zero error code to see if
 # verdi is installed properly
 script:
-    - sleep 30 # wait 30 seconds till the container is ready
-    - "export DOCKERID=`docker ps -qf 'ancestor=aiida-docker-stack'`"
     - "echo \"Docker ID: $DOCKERID\""
-    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c 'verdi status'"
+    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c '/usr/lib/postgresql/10/bin/pg_ctl -D /home/$SYSTEM_USER/.postgresql status'" # checking postgres
```

Would this also work?
If yes, then that would not break with the next postgres update
```suggestion
    - "docker exec --tty --user aiida \"$DOCKERID\" /bin/bash -l -c 'pg_ctl -D /home/$SYSTEM_USER/.postgresql status'" # checking postgres
```

---

unfortunately not:
```
aiida@7f63eea5893e:/$ pg_ctl
bash: pg_ctl: command not found
```